tests/languages/wgsl/operator_feature.test

```diff
@@ -0,0 +1,67 @@
+^
+~
+|
+||
+&&
+<<
+>>
+!
+&
++=
+-=
+*=
+/=
+%=
+^=
+&=
+|=
+<<=
+>>=
+=
+==
+!=
+<=
+>=
++
+%
+*
+-
+/
+.
```

Comment:
`.` should be punctuation. That's how we do it in pretty much every other C-like language, so let's keep it consistent.